content/en/admin/install.md

```diff
@@ -204,21 +201,33 @@ rm /etc/nginx/sites-enabled/default
 Then edit `/etc/nginx/sites-available/mastodon` to 
 
 1. Replace `example.com` with your own domain name
-2. Uncomment the `ssl_certificate` and `ssl_certificate_key` lines and replace the two lines with (ignore this step if you are bringing your own certificate)
+2. Uncomment the `ssl_certificate` and `ssl_certificate_key` (ignore this step if you are bringing your own certificate):
 
-```
-ssl_certificate     /etc/ssl/certs/ssl-cert-snakeoil.pem;
-ssl_certificate_key /etc/ssl/private/ssl-cert-snakeoil.key;
-```
+    ```
+    ssl_certificate     /etc/letsencrypt/live/example.com/fullchain.pem;
+    ssl_certificate_key /etc/letsencrypt/live/example.com/privkey.pem;;
+    ```
 
 3. Make any other adjustments you might need.
 
-Un-comment the lines starting with `ssl_certificate` and `ssl_certificate_key`, updating the path with the correct domain name.
+Add nginx's `www-data` user to the `mastodon` group to allow nginx to access asset files
+
+```bash
+usermod -aG mastodon www-data
+```
```

Comment:
I think I wouldn't do that, this may give write access to stuff we don't want. I'd just ensure there is proper traversal access instead.

Reply:
I linked a couple of issues where this was discussed before. Some proposed very generous permissions for `other` users. Using group permissions seemed to be the consensus for a more balanced approach.

> I'd just ensure there is proper traversal access instead.

But how exactly?

And why would that be sufficient? Our nginx config allows nginx to serve some static files directly. Does it not need read access for that to work?

Reply:
Isn't the issue the lack of traversal permission to `/home/mastodon`, or am I misunderstanding? `chmod o+x /home/mastodon` is sufficient to give traversal, and I think it's relatively safe

Reply:
This has been discussed in e.g. #1055 and the consensus seemed to be that a group based approach is tighter, because you do not need to give any kind of access to the mastodon home directory to _all_ other users on the system.

I can see things going wrong in both scenarios in theory and personally have no real preference here :man_shrugging:

Reply:
I think typically most files are writable by group? What do `ls -l /home`, `ls -l /home/mastodon` and `ls -l /home/mastodon/live` look like on a fresh 24.04 install?

Reply:
On Ubuntu all mastodon files are indeed writable for the group. On Debian they are not. Debian uses a stricter `umask` value compared to Ubuntu :weary: